include/dungeon_global_data.h

```diff
@@ -59,17 +71,17 @@ struct DungeonGlobalData
     s16 unkE23E;
     u8 fillE240[0xE264 - 0xE240];
     /* 0xE264 */ u8 weather; // Uses the weather constants in weather.h.
-    u8 unkE265;
+    u8 unkE265; // Uses the weather constants in weather.h
     /* 0xE266 */ u8 weatherDamageTimer; // Timer for applying sandstorm/hail damage periodically.
-    u8 fillE266[0xE26B - 0xE267];
+    u8 unkE267[0xE26B - 0xE267];
     u8 unkE26B;
     u8 weatherTurnsLeft;
-    u8 fillE26D[0xE270 - 0xE26D];
-    /* 0xE270 */ u8 startingWeather; // The weather at the start of the floor. If the weather changes, then expires, revert back to the starting weather.
-    u8 fillE271[0xE277 - 0xE271];
+    u8 fillE26D[0xE26F - 0xE26D];
+    /* 0xE26F */ u8 unkE26F[8]; // The weather at the start of the floor. If the weather changes, then expires, revert back to the starting weather.
```

Comment:
Perhaps this can still be called `startingWeather` until more is known about why it's an 8-element array.